Project_2/djangoProject/cars/models.py

```diff
@@ -5,19 +5,19 @@
 
 
 class Car(models.Model):
-    car_vin = models.CharField(max_length=17, unique=True,
-                               error_messages={"unique": "This car VIN already exists."},
-                               validators=[MinLengthValidator(17, "The car VIN has a 17 characters format!"),
-                                           RegexValidator(r'^[A-Z0-9]*$',
-                                                          'The car VIN has only numbers and capital letters!')])
+    vin = models.CharField(max_length=17, unique=True, default='',
```

Comment:
You don't need a default value here. The VIN either has the right format, or the car is not created at all. Also, there isn't any case where empty string is a possible value for a VIN.